public/app/plugins/datasource/loki/configuration/DerivedFields.tsx

```diff
@@ -91,7 +91,10 @@ export const DerivedFields = ({ fields = [], onChange }: Props) => {
             icon="plus"
             onClick={(event) => {
               event.preventDefault();
-              const newDerivedFields = [...fields, { name: '', matcherRegex: '', urlDisplayLabel: '', url: '' }];
+              const newDerivedFields = [
+                ...fields,
+                { name: '', matcherRegex: '', urlDisplayLabel: '', url: '', enableNameMatcher: false },
```

Comment:
Should this be `labelMatcher: 'true'`?

Reply:
Hmm, not sure, because that would technically break the default from "regex" to "label" and it might not be what users expect?